src/routes/Tool/VirtualAssistant.tsx

```diff
@@ -244,19 +245,26 @@ export const VirtualAssistant = ({
         </div>
       </div>
       {selectedKiinnostuksetViewVisible && (
-        <div id={selectedInterestsViewId} className="absolute top-0 w-full pt-6 h-full left-0 z-21">
-          <div className="bg-white rounded shadow-[0_-1px_24px_rgba(0,0,0,0.25)] h-full flex flex-col">
+        <div className="absolute top-0 w-full pt-6 h-full left-0 z-21">
+          <div
+            tabIndex={-1}
+            id={selectedInterestsViewId}
+            className="bg-white rounded shadow-[0_-1px_24px_rgba(0,0,0,0.25)] h-full flex flex-col"
+          >
             <button
               aria-label={t('tool.my-own-data.interests.virtual-assistant.close-selected-interests')}
-              onClick={() => setSelectedKiinnostuksetViewVisible(false)}
+              onClick={() => {
+                setSelectedKiinnostuksetViewVisible(false);
+                setTimeout(() => document.getElementById('kiinnostuksetView')?.focus(), 0);
+              }}
               className="absolute cursor-pointer self-end items-center p-4 m-3"
             >
               <span aria-hidden className="text-black sm:text-secondary-gray p-3 sm:p-0">
                 <MdClose size={24} />
               </span>
             </button>
             <div className="px-5 pt-9">
-              <h2 id={selectedKiinnostuksetLabelId} className="text-heading-4-mobile sm:text-heading-4 text-center">
```

Comment:
This `id={selectedKiinnostuksetLabelId}` should not be removed as it is used by the screenreader few lines below: 
```tsx
aria-labelledby={selectedKiinnostuksetLabelId}
```